builtin/providers/aws/resource_aws_api_gateway_domain_name.go

```diff
@@ -52,6 +52,11 @@ func resourceAwsApiGatewayDomainName() *schema.Resource {
 				Computed: true,
 			},
 
+			"certificate_upload_date": &schema.Schema{
```

Comment:
Hey - what i mean is, you have declared the parameter but  never set the parameter with a d.Set func call